acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -26,11 +26,12 @@
   end
 end
 
-if puppetdb_supported_platforms.include?(master.platform) then
-  step "Install PuppetDB repository" do
-    install_puppetlabs_dev_repo(
-      master, 'puppetdb', test_config[:puppetdb_build_version],
-      repo_config_dir, install_opts)
+puppetdb_supported_platforms.each do |pdb_platform|
+  if master.platform.include?pdb_platform.source then
```

Comment:
`master.platform` is a string, right?  Feels a little weird to me to do an `include?` on the original string since I think you'd lose the ability to match a regex.  Maybe this would be better?

~~~ruby
if master.platform ~= pdb_platform
~~~

?

Still wonder if it would be better to use beaker's `confine` and move the whole puppetdb install into a different test step rather than doing more seemingly hand-rolled iteration here?